toolkit/components/uniffi-bindgen-gecko-js/src/templates/Scaffolding.h

```diff
@@ -21,6 +23,21 @@ namespace mozilla::dom {
 
 class GlobalObject;
 
+{%- for object in ci.object_definitions() %}
+class {{ object.nm() }}PointerType : public UniFFIPointerType {
+    public:
+        static {{ object.nm() }}PointerType& getInstance() {
+            static {{ object.nm() }}PointerType instance;
+            return instance;
+        }
+    private:
+        {{ object.nm() }}PointerType() {
+            typeName = u"{{ ci.cpp_namespace() }}{{ object.nm() }}"_ns;
```

Comment:
I actually don't think the `typeName` is even needed anymore.. I just compare pointer values when checking if arguments are correct - I'll get rid of it in the next round after review

Reply:
I'm fine with getting rid of this, but maybe it would be nice for error formatting in some cases.

Reply:
Alright ya I'll keep it for now, it does no harm